src/qofonoextcell.cpp

```diff
@@ -452,6 +469,11 @@ int QOfonoExtCell::signalLevelDbm() const
     int QOfonoExtCell::x() const {\
         return Private::value(iPrivate, Private::Property_##x); \
     }
+#define PropertyGetU64_(x) \
+    quint64 QOfonoExtCell::x() const {\
+        return Private::value(iPrivate, Private::PropertyU64_##x); \
+    }
```

Comment:
`Private::value` returns an `int`. And `iProperties` is an int array too. We should store ints as int64, not the other way around.